assets/styles/components/_Boxes.scss

```diff
@@ -241,6 +241,12 @@
 
 		.wrapper .elementor-column {
 
+			transition: all 0.3s ease-in-out;
+
+			@include hover() {
```

Comment:
I would suggest to wrap :hover state in the mixin into @media (hover:hover) { &:hover} } media query